tests/Services/PedidosServiceTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Tests\Services;
+
+use \Cursos\Services\PedidosService;
+
+final class PedidosServiceTest extends \PHPUnit\Framework\TestCase {
+
+private $service;
+
+
+
+protected function setUp () : void {
+    $storage = new \Cursos\DB\MemoryStorage();
+    $this->service = new PedidosService($storage);
+} 
+
+public function testClassExist () {
+    $this->assertTrue(class_exists('Cursos\Services\PedidosService'));
+}
+
+public function testPuedeRegistrar () {
+
+    $pedido = $this->service->register('uno@uno.com','curso de macrame');
+    
+    $this->assertTrue($pedido instanceof \Cursos\Models\Pedido);
+}
+
+public function testPuedeRegistrarOtro () {
+    
+    $pedido = $this->service->register('uno@uno.com','curso de macrame');
+
+    $this->assertEquals('uno@uno.com', $pedido->getEmail());
+    $this->assertEquals('curso de macrame', $pedido->getTexto());
+}
+
+public function testPuedeRegistrarOtrosDatos () {
+
+    $pedido = $this->service->register('dos@dos.com','curso de macrame, la leyenda continua');
+
+    $this->assertEquals('dos@dos.com', $pedido->getEmail());
+    $this->assertEquals('curso de macrame, la leyenda continua', $pedido->getTexto());
+}
+
+public function testFindOne () {
+    
+    $pedido = $this->service->register('dos@dos.com','curso de macrame, la leyenda continua');
+
+    $found = $this->service->findOne($pedido->getId());
+
+    $this->assertTrue($found instanceof \Cursos\Models\Pedido);
```

Comment:
$this->assertEquals('uno@uno.com', $found->getEmail());
$this->assertEquals('curso de macrame, la leyenda continua', $found->getTexto());